test/cli/ArgumentParserTest.ooc

```diff
@@ -0,0 +1,97 @@
+use ooc-cli
+use ooc-collections
+use ooc-base
+use ooc-unit
+
+ArgumentParserTest: class extends Fixture {
+	init: func {
+		super("ArgumentParser")
+		this add("No parameter", func {
+			inputList := VectorList<Text> new()
+			inputList add(t"--arga")
+			inputList add(t"-b")
+			parser := ArgumentParser new()
+			valuea: Text
+			valueb: Text
+			parser add(t"arga", t"a", 0, Event new(func { valuea = t"a" }))
+			parser add(t"argb", t"b", 0, Event new(func { valueb = t"b" }))
+			parser parse(inputList)
+			expect(valuea, is equal to(t"a"))
+			expect(valueb, is equal to(t"b"))
+			inputList free()
+			parser free()
+			valuea free()
+			valueb free()
+		})
+		this add("With parameter", func {
+			inputList := VectorList<Text> new()
+			inputList add(t"--arga")
```

Comment:
Actually, `Fixture` doesn't support `is equal to()` with a `Text` argument. If it's not equal to, the whole test will crash. Better to use `==` for now.